doc/Release.md

```diff
@@ -37,7 +37,6 @@ This approach of having ~500 user on the nightly releases and ~5000 users on the
 - Do a quick check to make sure the build doesn't crash, e.g. by enrolling yourself into the test track.
     - In case there are any problems, delete the GitHub release, fix the problems and start again
 - Download the build as apk from Google Play (App Bundle Explorer -> choose the release -> Downloads -> Signed, universal APK). Attach it to the GitHub Release.
-- Update the download link in the [index.html of the website](https://github.com/tuskyapp/tuskyapp.github.io/blob/main/index.html) to point to the apk attached to the GitHub release.
```

Comment:
This is no longer necessary since we now just link to the latest release page on the homepage. https://github.com/tuskyapp/tuskyapp.github.io/pull/27